File: go/iq/waffle~get~disclosure+encryption_metadata+timestamp.go
// BuildWaffleGenerateWAEntACUser builds <iq type="get" xmlns="waffle">. Replies with one of: Success, Error.
func BuildWaffleGenerateWAEntACUser(smaxId int, id int, version string, lg string, lc string) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"xmlns": "waffle", "smax_id": smaxId, "to": "s.whatsapp.net", "type": "get"},
		Content: []waBinary.Node{
			{Tag: "encryption_metadata", Attrs: waBinary.Attrs{}},
			{Tag: "timestamp", Attrs: waBinary.Attrs{}},
			{Tag: "disclosure", Attrs: waBinary.Attrs{"id": id, "version": version, "lg": lg, "lc": lc}},
		},
	}
}

File: ts/iq/waffle~get~disclosure+encryption_metadata+timestamp.ts
/** <iq type="get" xmlns="waffle">. Replies with one of: Success, Error. */
export function buildWaffleGenerateWAEntACUser(smaxId: number, id: number, version: string, lg: string, lc: string): BinaryNode {
  return { tag: 'iq', attrs: { xmlns: 'waffle', smax_id: smaxId, to: 's.whatsapp.net', type: 'get' }, content: [
      { tag: 'encryption_metadata', attrs: {  } },
      { tag: 'timestamp', attrs: {  } },
      { tag: 'disclosure', attrs: { id: id, version: version, lg: lg, lc: lc } },
    ] }
}